analyzer/tests/unit/test_checker_handling.py

```diff
@@ -278,14 +330,18 @@ def test_disable_clangsa_checkers(self):
 
         for arg in analyzer.construct_analyzer_cmd(result_handler):
             if arg.startswith('-checks='):
-                self.assertIn('-clang-analyzer-*', arg)
+                self.assertLess(*self._enable_disable_pos(
```

Comment:
This is still a chore to digest. Can't you reduce it down to something easier to read? Like, `is_disabled`?